Testing/Dashboard/elxCommonCDash.cmake

```diff
@@ -0,0 +1,393 @@
+# ITK Common Dashboard Script
```

Comment:
"ITK Common Dashboard Script", should that be "_elastix_ Common Dashboard Script"? Or is it a copy from ITK? If it's a copy, could you please add a link to the original?

Reply:
@ViktorvdValk Thanks for adding the link to https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake !
Do you think it's necessary to keep maintaining our own copy of this file?  I'm somewhat concerned about future maintenance... 🤔

Reply:
I did change quite a lot in the file, so we should maintain our own copy I think.